packages/client/src/entity-actions/roster/addRosterEntry.tsx

```diff
@@ -45,18 +45,19 @@ export const addRosterEntry = new BulkAction(
     pii: false,
     refetchEntities: true,
   },
+  // eslint-disable-next-line require-await
   (async () => {
     return defer();
   }),
```

Comment:
We could avoid disabling eslint here by doing this instead. There are a couple other chunks like this I noticed that would be worth switching as well if you can.

```typescript
  () => {
    return Promise.resolve(defer());
  },
```

Reply:
Cool! Will implement